testsuite/integration/basic/pom.xml

```diff
@@ -185,6 +185,7 @@
                                         <include>org/jboss/as/test/integration/ee/injection/resource/basic/*TestCase.java</include>
                                         <include>org/jboss/as/test/integration/ee/naming/defaultbindings/**/*TestCase.java</include>
                                         <include>org/jboss/as/test/integration/transaction/inflow/TransactionInflowTestCase.java</include>
+                                        <include>org/jboss/as/test/integration/transaction/MaximumTimeoutTestCase.java</include>
```

Comment:
@tomjenkinson @zhfeng this also needs adding to the <exclude>s of the basic-integration-default-web.surefire execution

Reply:
done (I think) it was to add to exclude I think (the word you used before "s" is not showing in github UI correctly, maybe it is parsing some XML?)